3-JSS/src/modules/http.js

```diff
@@ -0,0 +1,28 @@
+import {config} from '../config.js'
+
+
+export class HttpClient {
```

Comment:
XMLHttpRequest is very very very old. fetch replaced it. but event with fetch, you should use `axios`

Reply:
create a facada called `serverApi.js` that will have function such as "getTodos" "addTodo"